packages/web/lib/logging/client.ts

```
@@ -134,7 +123,17 @@ export function createClientLogger() {
   return {
     error: (message: string, metadata?: LogMetadata) => {
       if (metadata?.error) {
-        metadata = { ...metadata, error: formatError(metadata.error) }
+        metadata = {
```

Biome (1.9.4)

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)